extensions/objects/templates/svg-filename-editor.component.ts

```diff
@@ -1192,32 +1212,28 @@ angular.module('oppia').component('svgFilenameEditor', {
         // This if condition is required to ensure that the size change is
         // applied only to the curve and not to all the control points.
         if (ctrl.drawMode === DRAW_MODE_BEZIER) {
-          var size = ctrl.fabricjsOptions.size;
-          var actualSize = parseInt(size.substring(0, size.length - 2));
           ctrl.canvas.getObjects().slice(-2).forEach(function(object) {
```

Comment:
Why slice(-2)? Maybe use a variable that represents what it is?

Reply:
Done.